docker/install/python.sh

```diff
@@ -24,4 +24,4 @@ apt-get update && apt-get install -y python3-dev
 # the version of the pip shipped with ubuntu may be too lower, install a recent version here
 cd /tmp && wget https://bootstrap.pypa.io/get-pip.py && python3 get-pip.py
 
-pip3 install pylint numpy requests Pillow pytest==5.3.2 pytest-env==0.6.2 pytest-cov==2.8.1 pytest-xdist==1.31.0
+pip3 install pylint numpy requests Pillow pytest==5.3.2 pytest-env==0.6.2 pytest-cov==2.8.1 pytest-xdist==1.31.0 pytest-faulthandler==2.0.1
```

Comment:
"This plugin is now part of pytest core since pytest 5.0, so users should not install this plugin together with that pytest version."  https://github.com/pytest-dev/pytest-faulthandler